extension/js/common/browser/browser-msg.ts

```diff
@@ -81,8 +82,12 @@ export namespace Bm {
   export type Ajax = { req: JQueryAjaxSettings; stack: string };
   export type AjaxGmailAttachmentGetChunk = { acctEmail: string; msgId: string; attachmentId: string };
   export type ShowAttachmentPreview = { iframeUrl: string };
+  export type ShowConfirmation = { text: string; isHTML: boolean; footer?: string };
   export type ReRenderRecipient = { email: string };
   export type SaveFetchedPubkeys = { email: string; pubkeys: string[] };
+  export type ProcessAndStoreKeysFromEkmLocally = { acctEmail: string; decryptedPrivateKeys: string[] };
```

Comment:
We removed these messages from `master`.
The merge is incorrect

Reply:
ProcessAndStoreKeysFromEkmLocally and GetLocalKeyExpiration

Reply:
thank you - i have removed it now from this PR.